requirements.txt

```diff
@@ -0,0 +1,60 @@
+flask==3.0.3 ; python_version >= "3.11" and python_version < "4.0"
+geopandas==0.14.3 ; python_version >= "3.11" and python_version < "4.0"
+greenlet==3.0.3 ; python_version >= "3.11" and python_version < "4.0" and (platform_machine == "aarch64" or platform_machine == "ppc64le" or platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64" or platform_machine == "win32" or platform_machine == "WIN32")
+idna==3.7 ; python_version >= "3.11" and python_version < "4.0"
+itsdangerous==2.1.2 ; python_version >= "3.11" and python_version < "4.0"
+jinja2==3.1.3 ; python_version >= "3.11" and python_version < "4.0"
+jmespath==1.0.1 ; python_version >= "3.11" and python_version < "4.0"
+kombu==5.3.7 ; python_version >= "3.11" and python_version < "4.0"
+mako==1.3.3 ; python_version >= "3.11" and python_version < "4.0"
+markupsafe==2.1.5 ; python_version >= "3.11" and python_version < "4.0"
+minio==7.2.5 ; python_version >= "3.11" and python_version < "4.0"
+numpy==1.26.4 ; python_version >= "3.11" and python_version < "4.0"
+packaging==24.0 ; python_version >= "3.11" and python_version < "4.0"
+pandas==2.2.2 ; python_version >= "3.11" and python_version < "4.0"
+prompt-toolkit==3.0.43 ; python_version >= "3.11" and python_version < "4.0"
+psycopg2-binary==2.9.9 ; python_version >= "3.11" and python_version < "4.0"
+pyarrow==15.0.2 ; python_version >= "3.11" and python_version < "4.0"
+pycparser==2.22 ; python_version >= "3.11" and python_version < "4.0"
+pycryptodome==3.20.0 ; python_version >= "3.11" and python_version < "4.0"
+pyjwt==2.8.0 ; python_version >= "3.11" and python_version < "4.0"
+pyproj==3.6.1 ; python_version >= "3.11" and python_version < "4.0"
+pyshp==2.3.1 ; python_version >= "3.11" and python_version < "4.0"
+python-dateutil==2.9.0.post0 ; python_version >= "3.11" and python_version < "4.0"
+python-dotenv==1.0.1 ; python_version >= "3.11" and python_version < "4.0"
+pytz==2024.1 ; python_version >= "3.11" and python_version < "4.0"
+pyyaml==6.0.1 ; python_version >= "3.11" and python_version < "4.0"
+redis==5.0.3 ; python_version >= "3.11" and python_version < "4.0"
+requests==2.31.0 ; python_version >= "3.11" and python_version < "4.0"
+s3transfer==0.10.1 ; python_version >= "3.11" and python_version < "4.0"
+shapely==2.0.3 ; python_version >= "3.11" and python_version < "4.0"
+six==1.16.0 ; python_version >= "3.11" and python_version < "4.0"
+sqlalchemy==2.0.29 ; python_version >= "3.11" and python_version < "4.0"
+typing-extensions==4.11.0 ; python_version >= "3.11" and python_version < "4.0"
+tzdata==2024.1 ; python_version >= "3.11" and python_version < "4.0"
+urllib3==2.2.1 ; python_version >= "3.11" and python_version < "4.0"
+vine==5.1.0 ; python_version >= "3.11" and python_version < "4.0"
+wcwidth==0.2.13 ; python_version >= "3.11" and python_version < "4.0"
+werkzeug==3.0.2 ; python_version >= "3.11" and python_version < "4.0"
```

Comment:
Yes, that's right. Also, I have this here in case we manually need to push the app. And we got this automatically generated on running `poetry export --without-hashes --format=requirements.txt > requirements.txt`.

Reply:
Yes, that's right. Also, we have this here in case we manually need to push the app. And we got this automatically generated on running poetry export --without-hashes --format=requirements.txt > requirements.txt.